en-US/firefox/products/developer-quantum.lang

```diff
@@ -39,6 +39,12 @@ New Tools
 Firefox DevTools
 
 
+## TAG: ember_reference_fix
+;The new Firefox DevTools are powerful, flexible, and best of all, hackable. This includes a best-in-class JavaScript debugger, which can target multiple browsers and is built in React and Ember.
```

Comment:
Remember to check the errors page, this would have raised an error. The reference string doesn't match the translation (which has correctly Redux instead of Ember)